src/mds/Mutation.h

```diff
@@ -161,7 +164,7 @@ inline ostream& operator<<(ostream &out, const MutationImpl &mut)
   return out;
 }
 
-typedef ceph::shared_ptr<MutationImpl> MutationRef;
+typedef boost::intrusive_ptr<MutationImpl> MutationRef;
```

Comment:
We probably want an alias in the ceph namespace for intrusive_ptr the same way we did for shared_ptr, so we can switch it around as they add it to the STL or use it with alternative systems.